how-tos/pub-sub/how-to.mdx

```diff
@@ -45,11 +45,7 @@ Click the **Subscribe** button on the basketball scoreboard to subscribe a clien
 
 The football scores are published using two different message names to differentiate between the green and blue leagues.
 
-Add the following parameters to `footballChannel.subscribe()` on L22 of `footballSubscriber.js`:
-
-```javascript
-footballLeague, (message)
-```
+Add the following parameters to `footballChannel.subscribe()` on L22 of `footballSubscriber.js`: `footballLeague`, `message`.
```

Comment:
@m-hulbert are you happy with the [syntax change here](https://ably-docs-next-staging.herokuapp.com/docs/how-to/pub-sub)?

![Sub How To 2024-12-04 16-56-43](https://github.com/user-attachments/assets/332dc3ec-5809-4156-a746-9927e95234ab)

Reply:
Mmm, that makes it technically incorrect as it's dropped the parentheses for starters. 

Seems an overly-aggressive linter, but at the very least it could sit as the following until we address MD more broadly:

```code
`footballLeague, (message)`
```

Reply:
Have updated with the above